tests/testthat/test-serializer-feather.R

```diff
@@ -1,7 +1,7 @@
 context("feather serializer")
 
 test_that("feather serializes properly", {
-  skip_if_not_installed("feather")
+  skip_if_not_installed("arrow")
 
   d <- data.frame(a=1, b=2, c="hi")
   val <- serializer_feather()(d, data.frame(), PlumberResponse$new(), stop)
```

Comment:
Can we change `application/feather` to `application/vnd.apache.arrow.file` in the lines below. Thank you!